### ISSUE TITLE
userActivity key caused crash on attempt to import

### ISSUE THREAD

**Issue by banj** [MarcDonald/obsidian-day-one-importer#5]
**Describe the bug**
I installed and enabled, then created day-one-in and day-one-out directories, and placed journal.json ready for import. Attempting to import caused immediate failure with a notification that the userActivity key had an undefined value.

**Comment by MarcDonald** [comment]
Thanks for the report! I can't seem to recreate this naturally by adding or removing an activity from a Day One entry and exporting it, but I think I can recreate it by manually adding a `userActivity` key with an empty object - is this the error you were seeing?

```
ZodError: [
  {
    "code": "invalid_type",
    "expected": "string",
    "received": "undefined",
    "path": [
      0,
      "userActivity",
      "activityName"
    ],
    "message": "Required"
  }
]
```

If so could you check your JSON for an empty `userActivity` key object? I have a fix for it but want to confirm that this is definitely the issue first

**Comment by banj** [comment]
Yep I think that's right, here's the console message:

```
plugin:day-one-importer:4050 ZodError: [
  {
    "code": "invalid_type",
    "expected": "string",
    "received": "undefined",
    "path": [
      1566,
      "userActivity",
      "activityName"
    ],
    "message": "Required"
  },
  {
    "code": "invalid_type",
    "expected": "string",
    "received": "undefined",
    "path": [
      1878,
      "userActivity",
      "activityName"
    ],
    "message": "Required"
  },
  {
    "code": "invalid_type",
    "expected": "string",
    "received": "undefined",
    "path": [
      1881,
      "userActivity",
      "activityName"
    ],
    "message": "Required"
  }
]
    at get error (plugin:day-one-importer:550:23)
    at ZodArray.parse (plugin:day-one-importer:654:18)
    at importJson (plugin:day-one-importer:4001:47)
    at async eval (plugin:day-one-importer:4225:23)
```

**Comment by MarcDonald** [comment]
This should be fixed in version 1.0.2, if you still have issues please re-open this ticket, thanks!
